src/deepsparse/server/README.md

```diff
@@ -0,0 +1,165 @@
+## DeepSparse Server 🔌
+
+```bash
+pip install deepsparse[server]
+```
+
+The DeepSparse inference server allows you to serve models and pipelines for deployment in HTTP. The server runs on top of the popular FastAPI web framework and Uvicorn web server. Currently, the server only supports NLP tasks, however support for computer vision will soon be released in upcoming versions!
```

Comment:
What's the formal name of this feature? DeepSparse Inference Server (akin to DeepSparse Engine)? e.g. trying to correlate the title of the page to the opening paragraph.

Reply:
```suggestion
The DeepSparse inference server allows you to serve models and pipelines for deployment in HTTP. The server runs on top of the popular FastAPI web framework and Uvicorn web server. Currently, the server only supports NLP tasks; however, support for computer vision will soon be released in upcoming versions.
```